app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+
+    command_split = command.split()
+    if (command_split[0] != "mv"
+            or command_split[1][-4:] != command_split[2][-4:]
+            or command_split[1][-4:] != ".txt"):
```

Comment:
bad implementation. Can you try to make another solution without such hardcoding?